src/Options.js

```diff
@@ -164,7 +165,7 @@ export class Options {
   }
 
   get sourceMaps() {
-    return this.sourceMaps_;
+    return this.lowResolutionSourceMap || this.sourceMaps_;
```

Comment:
Thought that maybe this option should implictly enable source maps. Soured on the idea though, lets scrap it.